### PR TITLE
[15.0][IMP] helpdesk_mgmt_sale: Allow to assign multiple tickets to an order

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT50683

### CODE REVIEW

**Comment by pilarvargas-tecnativa** [comment]
@pedrobaeza Do I include migration script?

**Comment by pedrobaeza** [comment]
Yes, please

File: helpdesk_mgmt_sale/migrations/15.0.1.0.0/post-migration.py
Comment (pedrobaeza) [inline helpdesk_mgmt_sale/migrations/15.0.1.0.0/post-migration.py:1]: Incorrect migration folder. You should put a higher one, and bump the version in the manifest.

File: helpdesk_mgmt_sale/models/helpdesk_ticket.py [hunk 1]
```diff
@@ -17,9 +17,9 @@ def _compute_so_count(self):
     def action_view_sale_orders(self):
         """Returns action to view sale orders related to this ticket."""
         action = self.env["ir.actions.actions"]._for_xml_id("sale.action_orders")
-        action["domain"] = [("ticket_id", "=", self.id)]
+        action["domain"] = [("ticket_ids", "in", [self.id])]
         action["context"] = {
-            "default_ticket_id": self.id,
+            "default_ticket_ids": [(6, 0, [self.id])],
```
Comment (pedrobaeza) [inline helpdesk_mgmt_sale/models/helpdesk_ticket.py]: I think this won't work

Comment (pilarvargas-tecnativa) [reply helpdesk_mgmt_sale/models/helpdesk_ticket.py]: why not?
ticket

Comment (pedrobaeza) [reply helpdesk_mgmt_sale/models/helpdesk_ticket.py]: For not losing existing data:

```suggestion
            "default_ticket_ids": [(4, self.id)],
```

Comment (pilarvargas-tecnativa) [reply helpdesk_mgmt_sale/models/helpdesk_ticket.py]: Actually with the tests I did, although the context is transmitted to the view it is only affecting when creating a new order and not the existing ones but you are right, the right way is to make sure that the existing ones are not overwritten. The change is done, thanks

**Review by pedrobaeza — APPROVED** [review]

**Review by victoralmau — APPROVED** [review]

**Pull request merged**